contracts/gateway/BridgeEscrow.sol

```diff
@@ -36,7 +36,6 @@ contract BridgeEscrow is Initializable, GraphUpgradeable, Managed {
      * @param _spender Address of the spender that will be revoked
      */
     function revokeAll(address _spender) external onlyGovernor {
```

Comment:
@tmigone do we have a test for calling revokeAll?

Reply:
yes we have one here: https://github.com/graphprotocol/contracts/blob/0f553c9d1ad68d626a8f1cbc44e9c24c32f45400/test/gateway/bridgeEscrow.test.ts#L67-L75